### ISSUE TITLE
How to set the cyclomatic complexity value when using gometalinter?

### ISSUE THREAD

**Issue by allencloud** [alecthomas/gometalinter#574]
Hi, All,

This is a question for the project. 

We know that when using gocyclo to calculates cyclomatic complexities of functions in Go source code, we can use `--over 5` flag to filter the functions which is over 5:

```shell
$ gocyclo -over 5 main.go
```

I am wondering if I could manually configure this flag `--over` for gocyclo in gometalinter.

also cc @fuweid

Thanks in advance

Allen

**Comment by allencloud** [comment]
I see that in the flags of gometalinter, there is a 
```
--cyclo-over=10          Report functions with cyclomatic complexity over N (using gocyclo).
```
It could solve my problem.
Closing this now.
